src/sctptransport.cpp

```diff
@@ -717,7 +717,7 @@ int SctpTransport::RecvCallback(struct socket *sock, union sctp_sockstore addr,
 	return ret;
 }
 
-int SctpTransport::SendCallback(struct socket *sock, uint32_t sb_free) {
+int SctpTransport::SendCallback(struct socket *sock, uint32_t sb_free, void *ulp_info) {
```

Comment:
You should not need to add this if you use the `usrsctp` version from `git submodule update`. Actually, I need to upgrade `usrsctp` and do some tests.